worlds/bk_sudoku/docs/fr_Sudoku.md

```diff
@@ -0,0 +1,13 @@
+# Bk Sudoku
+
+## Qu'est-ce que ce jeu ?
+
+BK Sudoku n'est pas un jeu typique d'Archipelago ; il s'agit plutôt d'un client générique de Sudoku qui peut se connecter à n'importe quel multi-monde existant. Une fois connecté, vous pouvez jouer au Sudoku pour débloquer des indices aléatoires pour votre jeu. Bien que lent, il vous donnera quelque chose à faire lorsque vous n'arrivez pas à atteindre les vérifications dans votre jeu.
+
+## Quels sont les indices débloqués ?
+
+Après avoir terminé un puzzle Sudoku, le jeu débloquera un indice aléatoire pour un emplacement non trouvé dans la partie à laquelle vous êtes connecté.
+
+## Où se trouve la page des options ?
+
+Il n'y a pas de page d'options ; ce jeu ne peut pas être utilisé dans vos .yamls. Au lieu de cela, le client peut se connecter à n'importe quel slot dans un multi-monde.
```

Comment:
"page d'options" feel wrong. Suggestion "page pour les options"

"slot" is changed with "partie" in the previous section. In this context, I believe the word "world" would be better, right? Thus, change "slot" to "monde".